packages/fiori/test/specs/NotificationList.spec.js

```diff
@@ -226,16 +226,34 @@ describe("Notification List Item Tests", () => {
 
 	it("tests Menu (actions / '...') button ACC attributes", async () => {
 		const firstItem = await browser.$("#nli1");
-		const menuButton = await firstItem.shadow$(".ui5-nli-menu-btn");
+		const menuButton = await firstItem.shadow$(".ui5-nli-menu-btn").shadow$("button");
+		const thirdItem = await browser.$("#nli3");
+		const btnListItemMenu3 = await thirdItem.shadow$(".ui5-nli-menu-btn");
 
 		// assert
-		assert.strictEqual(await menuButton.getAttribute("tooltip"), 'Actions', "The tooltip text is correct.");
+		assert.strictEqual(await menuButton.getAttribute("title"), 'Actions', "The tooltip text is correct.");
```

Comment:
Pls double check - the tooltip or the title.

Reply:
the 'tooltip' is the property, which adds 'title' attribute to the <button>
It depends which element is checked.